decouple.py

```diff
@@ -114,6 +120,9 @@ def __contains__(self, key):
     def __getitem__(self, key):
         return self.parser.get(self.SECTION, key)
 
+    def items(self):
+        return tuple(self.parser.items(self.SECTION))
```

Comment:
I think we could omit the tuple call.

Reply:
Oh, you mean we can return list both places. Ok, will change that.